packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -2974,6 +2974,7 @@
     "DIDDelete": 50,
     "OracleSet": 51,
     "OracleDelete": 52,
+    "NFTokenModify":53,
```

Comment:
Please use the xrpl-codec-gen script to auto-generate this instead of modifying the document by hand.

The ripple-binary-codec HISTORY.md will also need to be updated.